src/stores/SpaceStore.tsx

```diff
@@ -41,6 +41,9 @@ type SpaceKey = string | symbol;
 interface IState {}
 
 const ACTIVE_SPACE_LS_KEY = "mx_active_space";
+const LAST_VIEWED_ROOMS = "mx_last_viewed_rooms";
```

Comment:
can we turn this into a small method to get the LS key based on the given space as we have the same concat happening in two places e.g `getLastViewedRoomsStorageKey(space?)` or something more brief